### DISCUSSION TITLE
configuration update

### DISCUSSION THREAD

**Discussion by lidoror** [dani-garcia/vaultwarden#2222] [Q&A]
hi, I tried disabling the option to add the account by adding the value SIGNUPS_ALLOWED false in my Synology router but the changes didn't take effect and I had another problem when I changed my reverse proxy address the server didn't move to the new address.

**Comment by BlackDex** [comment]
You probably changed some settings via the /admin interface which invalidates environment variables. Check the /admin interface and there all dubble-underscored values have been set both via env and are different/changed via the /admin interfaces.

**Reply by lidoror** [reply]
how can i do it? i am new to this

**Reply by BlackDex** [reply]
See the wiki https://github.com/dani-garcia/vaultwarden/wiki in general.

And also check https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview
